mcaleb808/Taxi-24-APIs
# Configure database

---

#### Description of changes made

> Defined data models
> added SQL for creating table statements
> added data seeds


#### Type of change

> **Please select the relevant options**


#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/mcaleb808/Taxi-24-APIs), and `cd into the project`
- Check out this branch by _git checkout `ch-configure-database`_
- Run `yarn | npm install` to install the project dependencies
- Make sure to have your `.env` file set up correctly
- Run `yarn migrate:seeds | npm run migrate:seeds` to create tables and insert seeds.

#### Checklist:



#### Screenshots

|        BEFORE        |       AFTER       |
| :------------------: | :---------------: |
| Before these changes | After the changes |
